ddtrace/writer.py

```diff
@@ -107,21 +107,20 @@ def _log_error_status(self, response):
         if now > self._last_error_ts + LOG_ERR_INTERVAL:
             log_level = log.error
             self._last_error_ts = now
-        prefix = 'Failed to send traces to Datadog Agent at %s:%s: '
+        if self.api.uds_path:
+            prefix = 'Failed to send traces to Datadog Agent at %s: ' % self.api.uds_path
```

Comment:
Future TODO, we could just add `API.__repr__` and `'%s' % self.api` here instead.

Reply:
👍 I can do that in subsequent PR :)